arroyo/backends/kafka/configuration.py

```diff
@@ -49,6 +49,52 @@ def stats_callback(stats_json: str) -> None:
     )
 
 
+def producer_stats_callback(stats_json: str) -> None:
```

Comment:
Could we have a unit test for this? Just to ensure that if we make changes we do not make this crash the consumer by adding regressions.